code/src/nuvla/api/api.py

```diff
@@ -494,7 +492,7 @@ def edit(self, resource_id, data, **kwargs) -> CimiResource:
         :rtype:     CimiResource
         """
         resource = self.get(resource_id=resource_id)
-        operation_href = self._cimi_find_operation_href(resource, 'edit')
+        operation_href = resource.id
         return CimiResource(self._cimi_put(resource_id=operation_href, json=data, params=kwargs))
```

Comment:
IMO, having `operation_href` var is unnecessary. I suggest directly providing `resource.id` to the `resource_id` kwarg.